packages/api-client/src/libs/sendRequest.ts

```diff
@@ -93,18 +132,35 @@ export const sendRequest = async (
   } else if (example.body.activeBody === 'raw' && example.body.raw.value) {
     data = example.body.raw.value
   } else if (example.body.activeBody === 'formData') {
-    headers['Content-Type'] = 'multipart/form-data'
+    /**
+     * The header has to look something like this:
+     *
+     * Content-Type: multipart/form-data; boundary=----formdata-undici-043007900459
+     *
+     * fetch() makes sure to generate this properly, we must make sure to delete it
+     */
+    delete headers['Content-Type']
```

Comment:
```suggestion
    delete headers['Content-Type']
```
alright @hanspagel I think this is what we needed, try it now! 

Not sure if we want to delete this for everything or just formData

Reply:
I didn’t want to delete the header, because people should still be able to overwrite headers, e.g. send a wrong Content-Type header to test the behaviour of their API.

Is that fixing something specifically? Because the mentioned test, that is marked as failing, is still failing. 🤔

Reply:
Yea this fixes sending as formData because it adds the boundary at the end. Otherwise we might need to manually add the end boundary